triton-vm/src/table/processor_table.rs

```diff
@@ -413,6 +440,71 @@ impl ExtProcessorTable {
     }
 }
 
+#[derive(Debug, Copy, Clone, Display, EnumCountMacro, EnumIter, PartialEq, Hash)]
+pub enum ProcessorTableChallengeId {
+    /// The weight that combines two consecutive rows in the
+    /// permutation/evaluation column of the processor table.
+    StandardInputEvalIndeterminate,
+    StandardOutputEvalIndeterminate,
+    ToHashTableEvalIndeterminate,
+    FromHashTableEvalIndeterminate,
+
+    InstructionPermIndeterminate,
+    OpStackPermIndeterminate,
+    RamPermIndeterminate,
+    JumpStackPermIndeterminate,
+
+    /// Weights for condensing part of a row into a single column. (Related to processor table.)
+    InstructionTableIpWeight,
+    InstructionTableCiProcessorWeight,
+    InstructionTableNiaWeight,
+
+    OpStackTableClkWeight,
+    OpStackTableIb1Weight,
+    OpStackTableOspWeight,
+    OpStackTableOsvWeight,
+
+    RamTableClkWeight,
+    RamTableRamvWeight,
+    RamTableRampWeight,
+
+    JumpStackTableClkWeight,
+    JumpStackTableCiWeight,
+    JumpStackTableJspWeight,
+    JumpStackTableJsoWeight,
+    JumpStackTableJsdWeight,
+
+    UniqueClockJumpDifferencesEvalIndeterminate,
+    AllClockJumpDifferencesMultiPermIndeterminate,
+
+    // 2* Digest_Length elements of these
+    HashTableStackInputWeights0,
+    HashTableStackInputWeights1,
+    HashTableStackInputWeights2,
+    HashTableStackInputWeights3,
+    HashTableStackInputWeights4,
+    HashTableStackInputWeights5,
+    HashTableStackInputWeights6,
+    HashTableStackInputWeights7,
+    HashTableStackInputWeights8,
+    HashTableStackInputWeights9,
```

Comment:
What's with the plural identifier here? Isn't each of these one weight?

Reply:
Agreed.